docs/source/tutorials/01_assets/run_deformable_object.rst

```diff
@@ -0,0 +1,165 @@
+   :language: python
+   :start-at: # reset nodal state
+   :end-at: cube_object.reset()
+
+Stepping the simulation
+"""""""""""""""""""""""
+
+Deformable bodies support user-driven kinematic control where a user can specify position targets for some of
+the mesh nodes while the rest of the nodes are simulated using the FEM solver. This `partial kinematic`_ control
+is useful for applications where the user wants to interact with the deformable object in a controlled manner.
+
+In this tutorial, we apply kinematic commands to two out of the four cubes in the scene. We set the position
+targets for the node at index 0 (bottom-left corner) to move the cube along the z-axis.
+
+At every step, we increment the kinematic position target for the node by a small value. Additionally,
+we set the flag to indicate that the target is a kinematic target for that node in the simulation buffer.
+These are set into the simulation buffer by calling the :meth:`assets.DeformableObject.write_nodal_kinematic_target_to_sim`
+method.
+
+.. literalinclude:: ../../../../source/standalone/tutorials/01_assets/run_deformable_object.py
+   :language: python
+   :start-at: # update the kinematic target for cubes at index 0 and 3
+   :end-at: cube_object.write_nodal_kinematic_target_to_sim(nodal_kinematic_target)
+
+Similar to the rigid object and articulation, we perform the :meth:`assets.DeformableObject.write_data_to_sim` method
+before stepping the simulation. For deformable objects, this method does not apply any external forces to the object.
+However, we keep this method for completeness and future extensions.
+
+.. literalinclude:: ../../../../source/standalone/tutorials/01_assets/run_deformable_object.py
+   :language: python
+   :start-at: # write internal data to simulation
+   :end-at: cube_object.write_data_to_sim()
+
+Updating the state
+""""""""""""""""""
+
+After stepping the simulation, we update the internal buffers of the deformable object prims to reflect their new state
+inside the :class:`assets.DeformableObject.data` attribute. This is done using the :meth:`assets.DeformableObject.update` method.
+
+At a fixed interval, we print the root position of the deformable object to the terminal. As mentioned
+earlier, there is no concept of a root state for deformable objects. However, we compute the root position as
+the average position of all the nodes in the mesh.
+
+.. literalinclude:: ../../../../source/standalone/tutorials/01_assets/run_deformable_object.py
+   :language: python
+   :start-at: # update buffers
+   :end-at: print(f"Root position (in world): {cube_object.data.root_pos_w[:, :3]}")
+
+
+The Code Execution
+~~~~~~~~~~~~~~~~~~
+
+Now that we have gone through the code, let's run the script and see the result:
+
+.. code-block:: bash
+
+   ./isaaclab.sh -p source/standalone/tutorials/01_assets/run_deformable_object.py
+
+
+This should open a stage with a ground plane, lights, and several green cubes. Two of the four cubes must be dropping
```

Comment:
might be good to add a screenshot for the scene?

Reply:
Probably. We can do that in #827 for all of them together.